edgexfoundry/edgex-compose
# feat: support mediamtx rtsp server

This supports splitting out the mediamtx rtsp-server from the device-usb-camera service such that it can be made optional, as well as allows the end user to control the rtsp-server configuration in a more streamlined manner.







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions


```shell
make run rtsp-server ds-usb-camera
```

execute StartStreaming apis of device usb and verify streaming works
